### ISSUE TITLE
Exception thrown when callback function uses Python3 return value declaration

### ISSUE THREAD

**Issue by TheDiveO** [martinohanlon/BlueDot#67]
When I define a callback/event function as:
```python3
bd_pressed() -> None:
    ...
```
then bluedot/dot.py fails in `_process_callback` (or more precisely, in `getargspec`) with the error message "ValueError: Function has keyword-only arguments or annotations, use getfullargspec() API which can support them".

**Comment by martinohanlon** [comment]
Can you give me a little more information please.  

What are you trying to achieve?

Can you provide a full code listing of how to recreate the error as well please.

**Comment by martinohanlon** [comment]
Note - Migrate to use getfullargspec()

**Comment by TheDiveO** [comment]
```python
import bluedot
bd = bluedot.BlueDot()

def bd_pressed() -> None:
    pass

bd.when_pressed = bd_pressed
```

**Comment by martinohanlon** [comment]
Thanks

**Comment by martinohanlon** [comment]
fixed in v1.0.2
